tools/depends/target/Makefile

```diff
@@ -97,6 +97,7 @@ pythonmodule-pil: $(ZLIB) libjpeg-turbo libpng freetype2 python26
 libsdl2: $(LINUX_SYSTEM_LIBS)
 libxslt: libgcrypt
 ffmpeg: $(ICONV) $(ZLIB) bzip2 libvorbis $(FFMPEG_DEPENDS)
+libcec: platform
```

Comment:
@opdenkamp, I already started updating pvr add-ons. See argustv, demo, dvblink in kodi-pvr
If have others ready, I'll stop PR'ing. Just let me know.